packages/schema/docs/build/schema.md

```diff
@@ -1060,12 +1060,12 @@ How will we get a list of new objects? Will be turned into a trigger automatical
 
 * ```
   { display: { label: 'New User', description: 'Trigger when a new User is created in your account.' },
-    operation:
+    operation: 
```

Comment:
@eliangcs Noticed these got re-added again. Not sure the culprit, but I would guess it's tooling and not editor related (since I don't save/format this generated file in the editor ever). 

I've got `prettier -v => 1.18.2`, though I can't imagine prettier is _adding_ trailing spaces?